### PR TITLE
OCPBUGS#9904: Infra Node creation requires DNS follow-up actions.

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGS-9904

Link to docs preview:  https://59239--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating-infrastructure-machinesets.html#machineset-yaml-alibaba_creating-infrastructure-machinesets

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by xenolinux** [comment]
@lihongan Can you please take a look at this PR?

**Comment by lihongan** [comment]
@melvinjoseph86 could you please help take a look? 
And @sunilcio please help check as well since you're the QA contact of the bug. 
Thanks

File: modules/machineset-yaml-alibaba.adoc [hunk 1]
```diff
@@ -136,6 +136,11 @@ endif::infra[]
 <12> Specify the zone within your region to place machines on. Be sure that your region supports the zone that you specify.
 ifdef::infra[]
 <13> Specify a taint to prevent user workloads from being scheduled on infra nodes.
++
+[NOTE]
+====
+After adding the `NoSchedule` taint on the infrastructure node, existing pods running on that node are marked as `misscheduled`. You must either delete `misscheduled` pods or link:https://access.redhat.com/solutions/6592171[add toleration on `misscheduled` pods].
```
Comment (melvinjoseph86) [inline modules/machineset-yaml-alibaba.adoc:142]: i think it will dns pod on the node will be marked misscheduled not all pods

Comment (xenolinux) [reply modules/machineset-yaml-alibaba.adoc]: @melvinjoseph86 Addressed- can you please take a look now?

Comment (melvinjoseph86) [reply modules/machineset-yaml-alibaba.adoc]: looks better, thanks

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Review by sheriff-rh — COMMENTED** [review]
LGTM! Merging.

**Pull request merged**